LICENSE

```diff
@@ -25,4 +25,4 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The code in this package includes code adapted from WCSAxes, which is released
 under a 3-clause BSD license and can be found here:
 
-  https://github.com/astrofrog/wcsaxes
+  https://github.com/matplotlib/wcsaxes
```

Comment:
May I ask why? Won't it be confusing if some places point to https://github.com/astrofrog/wcsaxes and others to https://github.com/matplotlib/wcsaxes ?
Isn't https://github.com/matplotlib/wcsaxes the main location now?